src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Binskim.win-x86.RTR.dll.sarif

```diff
@@ -62,7 +62,7 @@
             "arguments": [
               "Binskim.win-x86.RTR.dll",
               "EnableSecureSourceCodeHashing",
-              "image is a managed IL library (i.e., ahead of time compiled) assembly"
+              "an exception occurred attempting to load its pdb"

```

Comment:
>"an exception occurred attempting to load its pdb" [](start = 14, length = 50)

this is wrong. a missing PDB is an error (because analysis that could have occurred did not). 'not applicable' means, 'we didn't look at this because the analysis is not applicable to this scan target'